fact_extractor/plugins/unpacking/uboot/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo "------------------------------------"
+echo "     install extract-dtb            "
+echo "------------------------------------"
+
+sudo pip3 install --upgrade extract-dtb
+exit 0
```

Comment:
`chmod +x`

😕 otherwise, the install fails

Reply:
Whoops :sweat_smile: should be executable now